Tools/CISupport/ews-build/steps.py

```diff
@@ -198,6 +200,18 @@ def _is_pr_blocked(self, pr_json):
                 return 1
         return 0
 
+    def _is_pr_in_mergequeue(self, pr_json):
+        for label in (pr_json or {}).get('labels', {}):
+            if label.get('name', '') == self.MERGE_QUEUE_LABEL:
+                return 1
+        return 0
+
+    def _is_pr_in_fastmergequeue(self, pr_json):
```

Comment:
Nit: _is_pr_in_fastmergequeue => _is_pr_in_fast_merge_queue

But do we need this separate function, this can be part of _is_pr_in_mergequeue.

Reply:
I'll combine them for now, but we probably will need something to differentiate at some point.